devito/finite_differences/derivative.py

```diff
@@ -135,7 +135,7 @@ def __new__(cls, expr, *dims, **kwargs):
         obj._dims = tuple(OrderedDict.fromkeys(new_dims))
         obj._fd_order = fd_orders
         obj._deriv_order = orders
-        obj._side = kwargs.get("side", centered)
+        obj._side = kwargs.get("side", None)
```

Comment:
`.get` assigns `None` automatically, ie it should be `kwargs.get("side")`

btw, do we actually need to default to `None`?

Reply:
Yes we need None